dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -65,45 +64,51 @@ public static void main(String[] args)
   }
 
   public static Path generate() throws IOException, TemplateException, ArtifactDescriptorException {
-    Configuration configuration = configure();
+    Configuration configuration = configureFreemarker();
     
     Path relativePath = Paths.get("target", "dashboard");
     Path output = Files.createDirectories(relativePath);
     
-    List<String> artifacts = readBom();
-    generateDashboard(configuration, output, artifacts);
-    generateReports(configuration, output, artifacts);
+    DefaultArtifact bom =
+        new DefaultArtifact("com.google.cloud:cloud-oss-bom:pom:0.62.0-SNAPSHOT");
+    List<Artifact> managedDependencies = readBom(bom);
+    
+    generateDashboard(configuration, output, managedDependencies);
+    generateReports(configuration, output, managedDependencies);
     
     return output;
   }
 
-  private static Configuration configure() {
+  private static Configuration configureFreemarker() {
```

Comment:
What is Freemarker?

Reply:
The template engine we use to generate the dashboard HTML.